llama-index-core/llama_index/core/llms/callbacks.py

```diff
@@ -148,7 +148,9 @@ async def wrapped_gen() -> ChatResponseAsyncGen:
         def wrapped_llm_chat(
             _self: Any, messages: Sequence[ChatMessage], **kwargs: Any
         ) -> Any:
-            with wrapper_logic(_self) as callback_manager:
+            with wrapper_logic(_self) as callback_manager, callback_manager.as_trace(
```

Comment:
What about the async wrappers though? Seems like those are missed

Reply:
ok，I will check all of them